app/assets/stylesheets/application.css.scss

```diff
@@ -46,7 +46,7 @@ h4 {
 p {
```

Comment:
I think assigning styles to all <p> tags may be a bad practice. I think it would be better to use a class and assign that class to the <p> tag instead.